apps/external-devices/device-descriptors/Yost_3-Space.json

```diff
@@ -0,0 +1,19 @@
+{
+    "deviceVendor": "Yost Labs (YEI)",
```

Comment:
but double check because according to the PrioVR kickstarter thing, there was some corporate upheaval akin to steve jobs getting kicked out of apple